Add Python 3.9 Support

This is a fairly easy task for anyone that has the time to handle it. Basically, we need to see if all of our unit tests work on Python 3.9 and update the TravisCI configuration within the code.

This is where the Python 3.9 version will live.
https://github.com/matrix-profile-foundation/matrixprofile/blob/master/.travis.yml#L14

I have a working venv under python 3.9.1 and tried running the tests (from the master), following the code in travis, but all the tests failed trying to import a module that does not exist:

```python
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_visualize.py:17: in <module>
    from matrixprofile.algorithms.stomp import stomp
matrixprofile/__init__.py:6: in <module>
    from matrixprofile.version import __version__, __version_info__
E   ModuleNotFoundError: No module named 'matrixprofile.version'
```

Is that module deprecated or not added yet? I'm happy to add it or update the tests. Let me know what's the plan.

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,6 +12,7 @@ python:
   - 3.6
   - 3.7
   - 3.8
+  - 3.9
 
 before_install:
   - |

--- a/README.rst
+++ b/README.rst
@@ -55,6 +55,7 @@ Currently, we support the following versions of Python:
 * 3.6
 * 3.7
 * 3.8
+* 3.9
 
 Python 2 is no longer supported. There are earlier versions of this library that support Python 2.
 

